dts/bindings/pinctrl/pincfg-node.yaml

```diff
@@ -0,0 +1,162 @@
+
+      bias-pull-pin-default:
+        required: false
+        type: boolean
+        description: use pin-default pull state.
+
+      drive-push-pull:
+        required: false
+        type: boolean
+        description: drive actively high and low
+
+      drive-open-drain:
+        required: false
+        type: boolean
+        description: drive with open drain (hardware AND).
+
+      drive-open-source:
+        required: false
+        type: boolean
+        description: drive with open source (hardware OR).
+
+      drive-strength:
+        required: false
+        type: int
+        description: sink or source at most X mA
+
+      drive-strength-microamp:
+        required: false
+        type: int
+        description: sink or source at most X uA
+
+      input-enable:
+        required: false
+        type: boolean
+        description: |
+          enable input on pin (no effect on output, such as enabling an input
+          buffer)
+
+      input-disable:
+        required: false
+        type: boolean
+        description: |
+          disable input on pin (no effect on output, such as disabling an input
+          buffer)
+
+      input-schmitt-enable:
+        required: false
+        type: boolean
+        description: enable schmitt-trigger mode
+
+      input-schmitt-disable:
+        required: false
+        type: boolean
+        description: disable schmitt-trigger mode
+
+      input-debounce:
+        required: false
+        type: int
+        description: |
+          Takes the debounce time in usec as argument or 0 to disable debouncing
+
+      power-source:
+        required: false
+        type: int
+        description: select between different power supplies
+
+      low-power-enable:
+        required: false
+        type: boolean
+        description: enable low power mode
+
+      low-power-disable:
+        required: false
+        type: boolean
+        description: disable low power mode
+
+      output-disable:
+        required: false
+        type: boolean
+        description: disable output on a pin (such as disable an output buffer)
+
+      output-enable:
+        required: false
+        type: boolean
+        description: |
+          enable output on a pin without actively driving it (such as enabling
+          an output buffer)
+
+      output-low:
+        required: false
+        type: boolean
+        description: set the pin to output mode with low level
+
+      output-high:
+        required: false
+        type: boolean
+        description: set the pin to output mode with high level
+
+      sleep-hardware-state:
+        required: false
+        type: boolean
+        description: |
+          indicate this is sleep related state which will be programmed into
+          the registers for the sleep state.
+
+      slew-rate:
+        required: false
+        type: int
+        description: set the slew rate
```

Comment:
```suggestion
        description: slew rate in <INSERT_UNITS_HERE>
```

V/μs ?

Reply:
At least for STM32 slew-rate is used as a kind of enum, see e.g. https://github.com/zephyrproject-rtos/hal_stm32/blob/master/dts/st/f4/stm32f401c(b-c)ux-pinctrl.dtsi#L111 It's weird, but Linux does the same (using `int` values 0-3)